squidDrone/tests/i2c/i2c_interface_tests.cpp

```diff
@@ -0,0 +1,139 @@
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "i2c.hpp"
+
+namespace{
+
+  class I2CInterfaceTests: public ::testing::Test{
```

Comment:
You are testing i2c not the interface. why is it called I2CInterfaceTests

Reply:
I also thought about this name, but it is more or less the I2C Interface, cause its the interface of the CPU to I2C Bus. But I renamed it to I2CTests.

Reply:
You named the class i2c and you are testing the class, so i2c_test is correct.